drivers/sensor/ak8975/ak8975.h

```diff
@@ -23,13 +23,13 @@
 #define AK8975_MEASURE_TIME_US		9000
 #define AK8975_MICRO_GAUSS_PER_BIT	3000
 
-#ifdef DT_INST_0_INVENSENSE_MPU9150_BASE_ADDRESS
-#if DT_INST_0_ASAHI_KASEI_AK8975_BASE_ADDRESS != 0x0C
+#if DT_INST_NODE_HAS_PROP(0, reg)
```

Comment:
This doesn't look right.  It seems to have lost something related to having support for an instance embedded in the mpu9150.

If that's intended I'd prefer to see this conversion pulled out to its own commit with an explanation, since the rest appear to be automated transformations.

Reply:
It wasn't intended, just a case the script didn't handle correctly.  Fixing it up.